### PR TITLE
VecMapping

### PR DESCRIPTION
**Pull request by kubaplas**
The new type of mapping that holds the length of the structure and allows querying elements at the index.

### CODE REVIEW

File: dao-contracts/src/voting/governance_voting/voting.rs [hunk 1]
```diff
@@ -119,15 +119,15 @@ impl Voting {
...
-            VotingType::Informal => self.informal_voting_quorum.as_usize(),
-            VotingType::Formal => self.formal_voting_quorum.as_usize(),
+            VotingType::Informal => self.informal_voting_quorum,
+            VotingType::Formal => self.formal_voting_quorum,
         }
     }
 
-    pub fn get_result(&self, voters_number: usize) -> VotingResult {
-        if self.get_quorum() > voters_number {
+    pub fn get_result(&self, voters_number: u32) -> VotingResult {
```
Comment (zie1ony) [inline dao-contracts/src/voting/governance_voting/voting.rs:129]: Bit weird that `get_quorum` returns `U256` but `votes_number` is `u32`.

**Review by zie1ony — APPROVED** [review]

File: dao-contracts/src/voting/governance_voting/voting.rs (hunk 1 above)
Comment (kubaplas) [reply dao-contracts/src/voting/governance_voting/voting.rs:129]: It simplifies the code. Quorum is connected to data saved in storage, thus U256. `voters_number` is just a len of a mapping vector.

Comment (kubaplas) [reply dao-contracts/src/voting/governance_voting/voting.rs:129]: I think that setting types to more realistic values can be done at the end.

Comment (zie1ony) [reply dao-contracts/src/voting/governance_voting/voting.rs:129]: So reduce quorums in the variable repo to `u32`, as it doesn't make sens to have it as `U256`.

**Pull request merged**